auth/src/main/java/com/firebase/ui/auth/ui/ResultCodes.java

```diff
@@ -1,20 +0,0 @@
-package com.firebase.ui.auth.ui;
-
-import com.firebase.ui.auth.AuthUI;
-
-/**
- * Result codes returned when using {@link AuthUI.SignInIntentBuilder#build()} with
- * {@code startActivityForResult}.
- */
-@Deprecated
-public class ResultCodes {
-
-    /**
-     * Sign in failed due to lack of network connection
-     *
-     * @deprecated Please use {@link com.firebase.ui.auth.ErrorCodes#NO_NETWORK}
-     **/
-    @Deprecated
-    public static final int RESULT_NO_NETWORK = com.firebase.ui.auth.ErrorCodes.NO_NETWORK;
```

Comment:
We should probably keep a running list of 1.x --> 2.x upgrade tips since something like this will be a little confusing.  Not sure what the best format for this is, maybe an MD file?